hack/install-pipeline-service.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -e
+
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
+
+source ${ROOT}/hack/flags.sh "The install-pipeline-service.sh installs Pipeline Service for development and testing on non-production clusters / kcp instances."
+MODE=${MODE:-preview} parse_flags $@
+
+
+PIPELINE_SERVICE_DIR=$(mktemp -d)
+git clone -b record-server-ca-during-registration --depth 1 https://github.com/lcarva/pipeline-service/ $PIPELINE_SERVICE_DIR
+export WORK_DIR="${PIPELINE_SERVICE_DIR}/gitops/sre/"
+export WORKSPACE_DIR=$WORK_DIR
+
+if [ "$ROOT_WORKSPACE" == "~" ]; then
```

Comment:
Should there me something early in this script that slurps in the values from `hack/preview.env` ?

Reply:
done by `parse_flags`